projects/assets/src/translations/en/product.ts

```diff
@@ -61,6 +61,8 @@ export const product = {
     reviewTitle: 'Review Title',
     writeYourComments: 'Write your comments',
     rating: 'Rating',
+    ariaLabelRating: 'Product rating, required',
+    ariaLabelRate: 'Add rate: {{rating}}',
```

Comment:
Hey @Matejk00 I think the names of these two keys can be changed to something more descriptive of their value rather than their purpose. What I mean is, instead of:
`ariaLabelRating: 'Product rating, required',`
It can be:
`ratingRequired: 'Product rating, required',`

Just something to think about